### ISSUE TITLE
Memory usage increasing with increase in available GPU memory

### ISSUE THREAD

**Issue by shekargoud26** [zhang0jhon/AttentionOCR#73]
I have run the provided docker image in 2 machines with 2 different GPUs
1. GTX 970m 4GB
2. Tesla V100 32GB

Its using around 220mb in 970m and 790mb on V100
#### here is the screenshot of 970m
![gtx970m](https://user-images.githubusercontent.com/16861024/82427644-ff7f4180-9aa6-11ea-9820-b6bd2ce2c48d.png)
#### here is the screenshot of V100
![v100](https://user-images.githubusercontent.com/16861024/82427731-1cb41000-9aa7-11ea-8745-be7d2390cf65.png)

both running on Ubuntu. Is there any way to fix the amount of GPU memory used by this api?

**Comment by encoding-muzzu** [comment]
yeah same here, even when i run in teslav100 , its occupying almost 8GB of memory, is there any way to reduce the memory?

![image](https://user-images.githubusercontent.com/20749757/82488718-fd949d00-9afd-11ea-8d32-0b25747221fa.png)
